iina/Base.lproj/MainMenu.xib

```diff
@@ -551,7 +551,7 @@ CA
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <menu key="submenu" title="Find Online Subtitles from" id="i0H-bE-bH8"/>
                             </menuItem>
-                            <menuItem title="Save Downloaded Subtitle" id="9zk-co-roT">
+                            <menuItem title="Save Downloaded Subtitle..." id="9zk-co-roT">
```

Comment:
Use ellipsis (…)

Reply:
Fixed in new commit.
Thanks for catching!